packages/annotations/src/store/selectors.js

```diff
@@ -4,6 +4,8 @@
 import createSelector from 'rememo';
 import { get, flatMap } from 'lodash';
 
+const emptyArray = [];
```

Comment:
Minor: I think in other places we called this `EMPTY_ARRAY`.

Reply:
Wouldn't be bad to have the same comment: https://github.com/WordPress/gutenberg/blob/563ef41c663f163a725c2ef7e961b11ea6981f6e/packages/editor/src/store/selectors.js#L57-L66

Reply:
> Minor: I think in other places we called this `EMPTY_ARRAY`.

Related guideline: https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#constants